rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -17,21 +17,20 @@
 
 Summary: Tasks support for Foreman with Dynflow integration
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.10.0
-Release: 2%{?foremandist}%{?dist}
+Version: 0.10.9
+Release: 1%{?foremandist}%{?dist}
 Group: Development/Libraries
 License: GPLv3
 URL: https://github.com/theforeman/foreman-tasks
 Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: foreman >= 1.15.0
 
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core)
-Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.22
+Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.29
```

Comment:
I think this also needs a package update:
```DEBUG util.py:439:  Error: No Package found for tfm-rubygem(dynflow) >= 0.8.29```

Reply:
This was supposed do be done already here https://github.com/theforeman/foreman-packaging/pull/1983. Is there anything else I have to do?

Reply:
Looks like I only released it to plugins (nonscl) and not core foreman (scl). I'll fix that later.